### PR TITLE
Vet Clinic Database Milestone: Query and update animals table

### PR DESCRIPTION
**Pull request by CarlosIgreda**
For this Milestone I:

✅ Used GitFlow
✅ Used database transactions
✅ Modified and deleted data in SQL
✅ Prepared complex queries that answer analytical questions

Screenshots:

✅ Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

image

image

✅ Inside a transaction:

- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- Verify that changes were made.
- Commit the transaction.
- Verify that changes persist after commit.

image

✅ Inside a transaction delete all records in the animals table, then roll back the transaction.

image

image

✅ Inside a transaction:

- Delete all animals born after Jan 1st, 2022.
- Create a savepoint for the transaction.
- Update all animals' weight to be their weight multiplied by -1.
- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Commit transaction

image

✅ Write queries to answer the following questions:

- How many animals are there?

image

- How many animals have never tried to escape?

image

- What is the average weight of animals?

image

- Who escapes the most, neutered or not neutered animals?

image

- What is the minimum and maximum weight of each type of animal?

image

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +97,7 @@ N/A.
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Query and update animals table**
+- [ ] **Query multiple tables**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (t-yanick) [inline README.md:103]: - [x] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @CarlosIgreda , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  SQL commands put in the correct files
:heavy_check_mark: Screenshots of transaction results and query results added to PR description.
:heavy_check_mark: Well documented PR
:heavy_check_mark: Use of correct Gitflow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CarlosIgreda) [reply README.md:103]: @t-yanick Done it. The license file was added.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @CarlosIgreda , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested change implemented.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
@t-yanick, thank you for your feedback.

**Pull request merged**